scipy/special/basic.py

```diff
@@ -754,20 +754,101 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse of the error function erf.
+
+    Computes the inverse of the error function.
```

Comment:
`erf` is a function from `C` to `C`, and this is a many-to-one mapping as indicated by `erf_zeros`. So this probably needs a little more explanation.  Perhaps something like:

"`erf` is a complex-valued function defined in the complex plane but restricting `erf` to the real line maps `[-infty, infty]` to `[-1, 1]` in a 1-1 manner.  `erfc` is the inverse to `erf`, defined on just the real interval `[-1, 1]`.  If presented a complex number, `erfinv` raises a TypeError.  If presented a real number outside `[-1, 1]`, it returns one of `+infty` or `-infty`."

Reply:
ok. I also feel to descript the `erf_zeros`.
I will revise the explanation of `erf_zeros` based on your comment.